man/journald.conf.xml

```diff
@@ -140,7 +140,13 @@
         following units: <literal>s</literal>, <literal>min</literal>,
         <literal>h</literal>, <literal>ms</literal>,
         <literal>us</literal>. To turn off any kind of rate limiting,
-        set either value to 0.</para></listitem>
+        set either value to 0.</para>
+
+        <para>If a service provides rate limits for itself through
+        <varname>LogRateLimitIntervalSec=, LogRateLimitBurst=</varname>
```

Comment:
hmm, please end the `<varname>` tag before the comma, and start a new one after. These are two stanzas after all, not one.